src/funcx_common/task_storage/s3.py

```diff
@@ -0,0 +1,50 @@
+import typing as t
+
+import boto3
```

Comment:
This imports `boto3`, but `boto3` isn't added to the dependencies for funcx-common.
I believe that it should be added in the same manner as the `redis` dependency: an extra which pulls in `boto3`, and a conditional import here. The extra can be `boto`, `aws`, or any other similar name.

Reply:
This should already be fixed.